pkg/windows/windows.go

```diff
@@ -960,6 +971,12 @@ func rmDirCmd(dirName string) string {
 	return fmt.Sprintf("if(Test-Path %s) {Remove-Item -Recurse -Force %s}", dirName, dirName)
 }
 
+// rmK8sDirCmd returns the PowerShell command to remove the k8sDir excluding WICD files
+func rmK8sDirCmd() string {
```

Comment:
The name is misleading as you are not removing the directory but just the non WICD files. I suggest naming this `rmK8sFilesCmd()` or `rmNonWICDFilesCmd()` something similar.

Reply:
updated to rmK8sFilesCmd()